sdb/commands/pyfilter.py

```diff
@@ -24,6 +24,9 @@
 
 
 class PyFilter(sdb.Command):
+    """
+    TODO
+    """
```

Comment:
same here

Reply:
I wasn't clear what the difference was between filter and pyfilter. Would the following work? Other examples you use on a regular basis?
```
Return objects matching the Python expression

EXAMPLES
    Print the even addresses

        sdb> echo 0 1 2 3 4 | cast int | pyfilter obj % 2 == 0
        (int)0
        (int)2
        (int)4
```

Reply:
I'd recommend we just leave this undocumented for now.

In general, I think folks should be using `filter`.. but I think there are more complex cases where `filter` won't work, which is when `pyfilter` could be used.. since the cases where `pyfilter` is required should be less common, I don't think it's critical to have this command documented (for now, at least).

Reply:
Removed